### PR TITLE
Don't log params of imagecreatefromstring

### PR DESCRIPTION
**Pull request by PVince81**
To prevent flooding the log with actual image data.

Found these in my logs of NC 20.0.2:
```
{
  "reqId": "Tr2WqCKLqkjqLVmQaQfq",
  "level": 3,
  "time": "2020-12-02T17:48:54+00:00",
  "remoteAddr": "77.190.205.171",
  "user": "vincent",
  "app": "PHP",
  "method": "GET",
  "url": "/index.php/apps/files/api/v1/thumbnail/256/256/favpics/dsc03277.jpg",
  "message": {
    "Exception": "Error",
    "Message": "imagecreatefromstring(): Data is not in a recognized format at /var/www/html/nextcloud/lib/private/legacy/OC_Image.php#624",
    "Code": 0,
    "Trace": [
      {
        "function": "onError",
        "class": "OC\\Log\\ErrorHandler",
        "type": "::",
        "args": [
          2,
          "imagecreatefromstring(): Data is not in a recognized format",
          "/var/www/html/nextcloud/lib/private/legacy/OC_Image.php",
          624,
          {
            "imagePath": "/mnt/data/nextcloud/vincent/files/favpics/dsc03277.jpg",
            "iType": false
          }
        ]
      },
      {
        "file": "/var/www/html/nextcloud/lib/private/legacy/OC_Image.php",
        "line": 624,
        "function": "imagecreatefromstring",
        "args": [
          "UYDAL4mwZf941QdkVy0dq6LK4............."
        ]
      },
      {
        "file": "/var/www/html/nextcloud/lib/private/Preview/Image.php",
        "line": 52,
        "function": "loadFromFile",
        "class": "OC_Image",
        "type": "->",
        "args": [
          "/mnt/data/nextcloud/vincent/files/favpics/dsc03277.jpg"
        ]
      },
      {
        "file": "/var/www/html/nextcloud/lib/private/Preview/GeneratorHelper.php",
        "line": 63,
        "function": "getThumbnail",
        "class": "OC\\Preview\\Image",
        "type": "->",
        "args": [
          {
            "__class__": "OC\\Files\\Node\\File"
          },
          4096,
          4096
        ]
      },
      {
        "file": "/var/www/html/nextcloud/lib/private/Preview/Generator.php",
        "line": 244,
        "function": "getThumbnail",
        "class": "OC\\Preview\\GeneratorHelper",
        "type": "->",
        "args": [
          {
            "__class__": "OC\\Preview\\JPEG"
          },
          {
            "__class__": "OC\\Files\\Node\\File"
          },
          4096,
          4096
        ]
      },
      {
        "file": "/var/www/html/nextcloud/lib/private/Preview/Generator.php",
        "line": 140,
        "function": "getMaxPreview",
        "class": "OC\\Preview\\Generator",
        "type": "->",
        "args": [
          {
            "__class__": "OC\\Files\\SimpleFS\\SimpleFolder"
          },
          {
            "__class__": "OC\\Files\\Node\\File"
          },
          "image/jpeg",
          ""
        ]
      },
      {
        "file": "/var/www/html/nextcloud/lib/private/Preview/Generator.php",
        "line": 109,
        "function": "generatePreviews",
        "class": "OC\\Preview\\Generator",
        "type": "->",
        "args": [
          {
            "__class__": "OC\\Files\\Node\\File"
          },
          [
            {
              "width": 256,
              "height": 256,
              "crop": true,
              "mode": "fill"
            }
          ],
          "image/jpeg"
        ]
      },
      {
        "file": "/var/www/html/nextcloud/lib/private/PreviewManager.php",
        "line": 190,
        "function": "getPreview",
        "class": "OC\\Preview\\Generator",
        "type": "->",
        "args": [
          {
            "__class__": "OC\\Files\\Node\\File"
          },
          256,
          256,
          true,
          "fill",
          null
        ]
      },
      {
        "file": "/var/www/html/nextcloud/apps/files/lib/Controller/ApiController.php",
        "line": 128,
        "function": "getPreview",
        "class": "OC\\PreviewManager",
        "type": "->",
        "args": [
          {
            "__class__": "OC\\Files\\Node\\File"
          },
          256,
          256,
          true
        ]
      },
      {
        "file": "/var/www/html/nextcloud/lib/private/AppFramework/Http/Dispatcher.php",
        "line": 169,
        "function": "getThumbnail",
        "class": "OCA\\Files\\Controller\\ApiController",
        "type": "->",
        "args": [
          256,
          256,
          {
            "__class__": "OC\\Files\\Node\\File"
          }
        ]
      },
      {
        "file": "/var/www/html/nextcloud/lib/private/AppFramework/Http/Dispatcher.php",
        "line": 100,
        "function": "executeController",
        "class": "OC\\AppFramework\\Http\\Dispatcher",
        "type": "->",
        "args": [
          {
            "__class__": "OCA\\Files\\Controller\\ApiController"
          },
          "getThumbnail"
        ]
      },
      {
        "file": "/var/www/html/nextcloud/lib/private/AppFramework/App.php",
        "line": 152,
        "function": "dispatch",
        "class": "OC\\AppFramework\\Http\\Dispatcher",
        "type": "->",
        "args": [
          {
            "__class__": "OCA\\Files\\Controller\\ApiController"
          },
          "getThumbnail"
        ]
      },
      {
        "file": "/var/www/html/nextcloud/lib/private/Route/Router.php",
        "line": 308,
        "function": "main",
        "class": "OC\\AppFramework\\App",
        "type": ": : ",
        "args": [
          "APIController",
          "getThumbnail",
          {
            "__class__": "OC\\AppFramework\\DependencyInjection\\DIContainer"
          },
          {
            "action": null,
            "x": "256",
            "y": "256",
            "file": "favpics/dsc03277.jpg",
            "_route": "files.API.getThumbnail"
          }
        ]
      },
      {
        "file": "/var/www/html/nextcloud/lib/base.php",
        "line": 1009,
        "function": "match",
        "class": "OC\\Route\\Router",
        "type": "->",
        "args": [
          "/apps/files/api/v1/thumbnail/256/256/favpics/dsc03277.jpg"
        ]
      },
      {
        "file": "/var/www/html/nextcloud/index.php",
        "line": 37,
        "function": "handleRequest",
        "class": "OC",
        "type": ": : ",
        "args": []
      }
    ],
    "File": "/var/www/html/nextcloud/lib/private/Log/ErrorHandler.php",
    "Line": 91,
    "CustomMessage": "--"
  },
  "userAgent": "Mozilla/5.0(Android)Nextcloud-android/3.14.1",
  "version": "20.0.1.1"
}
```

### CODE REVIEW

File: lib/private/Log/ExceptionSerializer.php [hunk 1]
```diff
@@ -86,6 +86,9 @@ class ExceptionSerializer {
 
 		// files_external: UserStoragesController
 		'update',
+
+		// Preview providers, don't log big data strings
+		//'imagecreatefromstring',
```
Comment (ChristophWurst) [inline lib/private/Log/ExceptionSerializer.php:91]: :ghost:

Comment (PVince81) [reply lib/private/Log/ExceptionSerializer.php:91]: yep... just wanted to quickly cancel CI but have no perms :'(

**Comment by PVince81** [comment]
rebased

**Review by ChristophWurst — APPROVED** [review]

**Review by MorrisJobke — APPROVED** [review]

**Pull request merged**